clouddq/classes/dq_configs_cache.py

```diff
@@ -461,42 +461,38 @@ def _resolve_bigquery_entity_uri(
                 "metadata_default_registries YAML configs, e.g.\n"
                 f"{SAMPLE_DEFAULT_REGISTRIES_YAML}"
             )
-        required_arguments = ["projects", "lakes", "locations", "zones"]
+        required_arguments = ["projects", "datasets", "tables"]
         for argument in required_arguments:
             uri_argument = entity_uri.get_configs(argument)
             if not uri_argument:
                 raise RuntimeError(
-                    f"Failed to retrieve default Dataplex '{argument}' for "
+                    f"Failed to retrieve default Bigquery '{argument}' for "
                     f"entity_uri: {entity_uri.complete_uri_string}. \n"
                     f"'{argument}' is a required argument to look-up metadata for the entity_uri "
-                    "using Dataplex Metadata API.\n"
-                    "Ensure the BigQuery dataset containing this table "
-                    "is registered as an asset in Dataplex.\n"
-                    "You can then specify the corresponding Dataplex "
-                    "projects/locations/lakes/zones as part of the "
-                    "metadata_default_registries YAML configs, e.g.\n"
-                    f"{SAMPLE_DEFAULT_REGISTRIES_YAML}"
+                    "using Bigquery API.\n"
+                    "Ensure the BigQuery dataset contains this table."
```

Comment:
Remove this sentence? This seems a bit unrelated if the error is about missing "project" or "datasets".

Reply:
done